introduction_to_amazon_algorithms/forecasting_services_comparison/SageMaker_DeepAR_Amazon_Forecast.ipynb

```diff
@@ -0,0 +1,1938 @@
+{
```

Comment:
<p>It would be more clear if each line output was a bit different like show the filename or just say "time series 1", "time series 2", etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3523/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>